lib/ClimaLandSimulations/src/utilities/climaland_output_dataframe.jl

```diff
@@ -28,7 +28,7 @@ end
 
 Return a dataframe containing climaland outputs
 """
-function make_output_df(sv, inputs)
+function make_output_df(sv, inputs; N_days = 60, N_spinup_days = 30)
```

Comment:
do we also need the state `u` and not just `sv`?

Reply:
we could return `u` in `fluxnet_simulation()`

Reply:
ok! we also want to plot the prognostic variables. so that sounds good!